docs/tutorials/running_on_lfrqa.md

```diff
@@ -0,0 +1,290 @@
+# Preparing the LFRQA Dataset
+
+## Overview
+
+The **LFRQA dataset** was introduced in the paper [_LFRQA: Large-Scale Few-Shot Retrieval Question Answering_](https://arxiv.org/pdf/2407.13998). It features **1,404 science questions** (along with other categories) that have been human-annotated with answers. This tutorial walks through the process of setting up the dataset for use and benchmarking.
+
+## Download the Annotations
+
+First, we need to obtain the annotated dataset from the official repository:
+
+```bash
+# Clone the RAG-QA Arena repository
+git clone https://github.com/awslabs/rag-qa-arena
+
+# Create a new directory for the dataset
+mkdir -p data/rag-qa-benchmarking
+
+# Move the science annotations file to our working directory
+mv rag-qa-arena/data/annotations_science_with_citation.jsonl ./data/rag-qa-benchmarking/
```

Comment:
If we just need one specific JSONL from GitHub, can we just do follow something like https://stackoverflow.com/a/4605068

And `curl` the specific file, as opposed to:
1. Cloning the full repo
2. `mv`'ing a specific file
3. `rm`'ing the rest of the repo